source/_elephpants/2022-06-14-cmgt.md

```diff
@@ -0,0 +1,15 @@
+---
+categories:
+    - original
+tags:
+    - green
```

Comment:
We have a "teal" group, including this one: https://afieldguidetoelephpants.net/#e-zend-teal-1445212800
Do you think that would be closer in color?

Reply:
Yes, teal is better than green. 

Also, I have now this elephpant in my possession, and I know the PHPather. We can validate this.